content/advanced/330_servicemesh_using_appmesh/cleanup/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Cleanup"
+date: 2020-01-27T08:30:11-07:00
+weight: 100
+draft: false
+---
+
+#### Delete all the resources in the namespace
+
+```bash
+kubectl delete all --all -n prodcatalog-ns
```

Comment:
Deleting the namespace (next step) will delete all the objects in the ns, should be fine to just do that one step.

Reply:
Updated